src/main/java/org/skriptlang/skript/lang/experiment/types/SingularExperimental.java

```diff
@@ -0,0 +1,24 @@
+package org.skriptlang.skript.lang.experiment.types;
+
+import ch.njol.skript.lang.SyntaxElement;
+import org.skriptlang.skript.lang.experiment.Experiment;
+import org.skriptlang.skript.lang.experiment.ExperimentData;
+import org.skriptlang.skript.lang.experiment.ExperimentalSyntax;
+
+/**
+ * Interface for expressing a {@link SyntaxElement} is an {@link ExperimentalSyntax} and requires only one
+ * {@link Experiment} to be enabled.
+ */
+public interface SingularExperimental extends ExperimentalSyntax {
```

Comment:
```suggestion
public interface SingularExperimentSyntax extends ExperimentalSyntax {
```
seems like a better naming scheme imo